.travis.yml

```diff
@@ -1 +1,27 @@
 language: rust
+rust:
+  - stable
+  - nightly
+
+os:
+  - linux
+
+dist: bionic
+
+env:
+  global:
+    - RUST_BACKTRACE=1
+
+addons:
+  apt:
+    packages:
+      - clang
+      - libjudy-dev
+      - libarchive-dev
+      - pkg-config
+      - build-essential
+      - m4
+
+before_install:
+  - git clone https://github.com/traildb/traildb /tmp/traildb
+  - pushd /tmp/traildb && curl -o ./waf https://waf.io/waf-2.0.18 && chmod +x ./waf && ./waf configure && ./waf build && sudo ./waf install && sudo ldconfig && popd
```

Comment:
I'd move the waf version into an isolated environment variable, but it's probably a thing of taste.

Reply:
good idea, pushed additional commit